test/selenium_suite/src/edu/westmont/wicl/SampleTests.java

```diff
@@ -146,14 +146,29 @@ public void testFindFood() {
         // Now submit the form. WebDriver will find the form for us from the element
         //element.submit();
 
-        // Wait for the page to load, timeout after 10 seconds
-        //(new WebDriverWait(driver, 10)).until(new ExpectedCondition<Boolean>() {
-        //    public Boolean apply(WebDriver d) {
-        //       return d.getTitle().toLowerCase().startsWith("cheese!");
-        //    }
-        //});
-        
+     // Wait for the page to load, timeout after 10 seconds
+		(new WebDriverWait(driver, 10)).until(new ExpectedCondition<Boolean>() {
+			public Boolean apply(WebDriver d) {
+				System.err.println(d.getTitle());
+				return d.getTitle().toLowerCase().startsWith("Ally");
```

Comment:
This is never going to pass because you are turning the title to lower case then testing it against an upper case Ally.